modules/widgets/flickr/widget.php

```diff
@@ -0,0 +1,33 @@
+<!-- Start of Flickr Widget -->
+<table
```

Comment:
Why the `table` layout? Would it be possible to use regular divs, so folks can customize the layout with a bit less trouble?